### DISCUSSION TITLE
[Question] Getting rid off unecessary Logistic modules

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#13723] [How to]
Hi guys, seeking for advice.
I have basis needs to manage stock of little bakery.
1. Products production
2. Delivery to Customes
3. Taking old product out of Customer's stock

What modules in your opinion would be essential to keep eye on products flow in such basic case?
Appreciate sharing your inside

**Comment by vovpff** [comment]
@IanMuller IGDN, IGRN, IIDN. This is short names of modules.

**Comment by waran70** [comment]
IGDN? IGRN? I can decipher it of course but thhere is no such modules we can match this abbreviations precisely, or I am wrong?

**Comment by vovpff** [comment]
@waran70 you are wrong. Goods Dispatched Notes and Goods Receiver Notes.
![image](https://user-images.githubusercontent.com/8545791/97955516-696e7780-1db7-11eb-8149-e92ca94eab0e.png)

**Comment by waran70** [comment]
:-) --> "no such modules we can match this abbreviations precisely"

I meant it should be GDN, GRN...
